### PR TITLE
Enable bitcode for iOS builds by default

### LINKED ISSUE #5611: Is there any way to let TensorFlow support Bitcode on iOS?
I'm trying to use TensorFlow on a iOS App, but my team required Bitcode enabled in the project, so I seek for help here.

### PR DESCRIPTION
**Pull request by eaigner**
Closes #5611

### CODE REVIEW

File: tensorflow/contrib/makefile/build_all_ios.sh [hunk 1]
```diff
@@ -32,6 +32,13 @@ cd ${SCRIPT_DIR}/../../../
 make -f tensorflow/contrib/makefile/Makefile clean
 rm -rf tensorflow/contrib/makefile/downloads
 
+# Setting a deployment target is required for building with bitcode,
+# otherwise linking will fail with:
+#
+#    ld: -bind_at_load and -bitcode_bundle (Xcode setting ENABLE_BITCODE=YES) cannot be used together
+#
+export MACOSX_DEPLOYMENT_TARGET="10.10"
```
Comment (drpngx) [inline tensorflow/contrib/makefile/build_all_ios.sh]: Is that the lowest version we can support?

@petewarden just in case.

Comment (petewarden) [inline tensorflow/contrib/makefile/build_all_ios.sh]: Yosemite is pretty old at this point, so it seems a reasonable default.

**Review by drpngx — APPROVED** [review]
Thanks Pete!

**Comment by drpngx** [comment]
Jenkins, test this please.

**Pull request merged**